examples/wine_quality/model_defaults_example.ipynb

```diff
@@ -0,0 +1,299 @@
+{
```

Comment:
This seems potentially more useful/readable as a live colab link under docs, like the example notebook for [text classification](https://ludwig-ai.github.io/ludwig-docs/0.5/examples/text_classification/). WDYT?

Reply:
@justinxzhao Definitely agree. Would it make sense to have it in both places? I saw that we did something similar for hyperopt so I figured I'd replicate it. Happy to do whatever you think makes the most sense!

Reply:
Having this available in multiple places SGTM, though I'm not sure about the best way to keep the copies in sync. Maybe we can think about that later when we consolidate all of the "examples" we have across docs and the ludwig repo.

Reply:
Awesome, that sounds good. Happy to revisit when we're thinking of consolidating :)